docs/source/tools/sandbox.rst

```diff
@@ -132,39 +133,11 @@ use one of the following command line options:
   - ``--auth-jwt-hs256-unsafe=<secret>``.
     The sandbox will expect all tokens to be signed with HMAC256 with the given plaintext secret.
 
-Token payload
-=============
-
-JWTs express claims which are documented in the :ref:`authorization <authorization-claims>` documentation.
-
-The following is an example of a valid JWT payload:
-
-.. code-block:: json
-
-   {
-      "https://daml.com/ledger-api": {
-        "ledgerId": "aaaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee",
-        "participantId": null,
-        "applicationId": null,
-        "admin": true,
-        "actAs": ["Alice"],
-        "readAs": ["Bob"]
-      },
-      "exp": 1300819380
-   }
-
-where
-
-- ``ledgerId``, ``participantId``, ``applicationId`` restricts the validity of the token to the given ledger, participant, or application
-- ``exp`` is the standard JWT expiration date (in seconds since EPOCH)
-- ``admin``, ``actAs`` and ``readAs`` bear the same meaning as in the :ref:`authorization <authorization-claims>` documentation
-
-The ``public`` claim is implicitly held by anyone bearing a valid JWT (even without being an admin or being able to act or read on behalf of any party).
 
 Generating JSON Web Tokens (JWT)
```

Comment:
Wondering whether this would better sit on the auth page rather than the Sandbox page.

Reply:
I've added a corresponding note to the authorization.rst page, but left this section on the sandbox page to keep the flow.